### DISCUSSION TITLE
Mirror of this repository

### DISCUSSION THREAD

**Discussion by Ronsor** [ggml-org/llama.cpp#435] [Show and tell]
Since Facebook/Meta has started sending false DMCAs, I set up a mirror of this repo on my own git server: https://git.as211233.net/ronsor/llama.cpp

**Comment by Green-Sky** [comment]
they did? proof?

**Reply by j-f1** [reply]
https://github.com/shawwn/llama-dl

**Reply by Green-Sky** [reply]
well, this was to be expected.

**Comment by ggerganov** [comment]
So I think we should be more strict and erase comments posting links to torrent files or IPFS models

**Reply by Green-Sky** [reply]
i hope we can keep the checksums, at least for the converted files.

**Reply by gjmulder** [reply]
I asked ChatGPT:

_A SHA checksum of some data would not be liable for a DMCA takedown notice by itself. A SHA checksum is simply a unique digital fingerprint of the data, and it does not contain any actual content. Therefore, it does not infringe on any copyright or intellectual property rights._

_However, if the data itself is protected by copyright and is being distributed without authorization, the copyright owner may issue a DMCA takedown notice to the entity hosting or sharing the data, regardless of whether a SHA checksum is involved or not. In such cases, the SHA checksum may be used as evidence to confirm the presence of the infringing data._

**Reply by gjmulder** [reply]
Also:

_Contributory infringement occurs when a person knowingly provides substantial assistance or encouragement to someone else who is directly infringing on a copyright. In the context of linking to infringing content, contributory infringement would occur if the person linking to the content knew or had reason to know that the content was infringing and still provided a link to it._

_It's important to note that the laws surrounding DMCA takedown notices and copyright infringement can be complex and vary by jurisdiction. If you have specific concerns about linking to data that is restricted by a license, you should consult with a legal professional who can provide guidance based on your specific situation._

**Reply by ggerganov** [reply]
Checksums are totally fine

**Reply by Piezoid** [reply]
Magnet and IPFS links are checksums from a technical perspective, but the intention behind their use can make the difference, I believe.

**Reply by Green-Sky** [reply]
they are even checksums of checksums. afaik there are _systems_ that only use single hashes, but those are not too popular (thankfully)
